### PR TITLE
Fix CI Failures

### PR DESCRIPTION
**Pull request by loay**
- no test case is needed
- In `imported.test.js`, test cases are failing.
- This PR fixes issue and test cases pass.

### CODE REVIEW

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
@superkhau PTAL. Thanks.

**Comment by loay** [comment]
connect to https://github.com/strongloop/loopback-connector-mssql/issues/90

File: lib/migration.js [hunk 1]
```diff
@@ -333,7 +333,7 @@ function mixinMigration(MsSQL) {
         } else if (idProp.type === String) {
           if (idProp.generated !== false) {
             sql.push(self.columnEscaped(model, modelPKID) +
-              ' [uniqueidentifier] DEFAULT newid() NOT NULL');
+              ' [varchar](128) DEFAULT newid() NOT NULL');
```
Comment (raymondfeng) [inline lib/migration.js]: I don't think this the correct fix for a few reasons:

1. The proposed change is breaking compatibility
2. Please note `newid()` returns `uniqueidentifier`
3. The test probably assumes id is always a string 

We probably need to fix the test case instead.

**Comment by loay** [comment]
I will have to change the test case of  ```it('updates specific instances when PK is not an auto-generated id', function(done) {```
in `oopback-datasource-juggler/test/manipulation.test.js`

**Comment by superkhau** [comment]
@raymondfeng I'm reassigning this to you for review as you know more about this specific area than I do.

**Comment by loay** [comment]
@slnode test please

**Pull request closed without merging**